### PR TITLE
docs(release-notes): 1.4 release updates

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

Updates for the 1.4 release

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -2,51 +2,34 @@
...
 
 ### Additions
 #### RPS & MPS
-- **Stateless Authentication:** The RESTful APIs for both RPS and MPS had their APIKEYS removed and now leverage [Kong Gateway](https://konghq.com/kong/) for user authentication.  This simplifies how API calls are made into the Open AMT Cloud Toolkit services by routing all API connections through a single URL:   
-    Format:
-    ```html
-    https://example.url/
-    ```
-    Examples:
+- **Containerized Scaling Support:** In 1.2 we released a preview version of scaling.  This was a highly valuable exercise to learn how best to handle large scale deployments.  In 1.4 we are releasing our updated scaling capabilities with support for Docker Swarm and Kubernetes deployments.  This scaling feature will allow customers to scale out the MPS and RPS service as wide as needed to support their entire fleet of Intel&reg; AMT devices.  The following is a summary of some of the changes we have made to enable robust scaling:
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:10]: **Containerized Scaling Support:** In 1.2 we released a preview version of scaling, a first look at supporting large-scale deployments.  In 1.4 we've updated scaling capabilities with support for Docker Swarm and Kubernetes deployments.  
With this feature, scale the MPS and RPS microservices as needed to support an entire fleet of Intel&reg; AMT devices.  

The toolkit enables robust scaling through:

 * A new component (MPS Router) that acts as a reverse proxy between the API Gateway and the MPS.  More on MPS Router below.   
 *  Support for deploying cloud based components and dependencies to Docker Swarm and Kubernetes. Components include MPS, RPS, MPS Router, API Gateway, Vault, PostgreSQL.    
 * Cloud provider agnostic [deployment documentation](https://open-amt-cloud-toolkit.github.io/docs/1.4/Kubernetes/kubernetes/).     
 * Improved routing of KVM and SOL sessions when multiple instances of MPS are deployed.    
 * An increase in the number of  CIRA connections that can be handled per MPS instance.  Detailed guidance will be provided with the LTS release.



On that second bullet, I was really confused about the parens. I endeavored to remove the parens.

File: docs/release-notes.md [hunk 2]
```diff
@@ -2,51 +2,34 @@
...
 
+### Modifications and Removals
+#### MPS
+- **AllowList:** As noted in the 1.3 release notes, the AllowList was not working as designed.  What it was suppose to do is to give the ability to only allow specified devices (GUIDs) to connect to MPS.  When we removed "developer mode" this broke the AllowList functionality.  In this release we have updated MPS to only allow devices to connect if they are listed in the MPS database and present the proper credentials.  If a device either is not listed in the MPS database or has the wrong credentials, MPS will refuse the connection.  Along with this change, we have improved the MPS Devices API to make it very easy to add, edit, and remove devices from the MPS database.  MPS API documentation can be found out on [Swagger](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0#/)
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:23]: - **AllowList:** Allowlist enables administrators to specify which devices (GUIDs) can connect to MPS. Removing "developer mode" in a previous version broke Allowlist functionality. In this release, MPS AllowList functionality allows devices to connect if they are listed in the MPS database and present the proper credentials.  If a device either is not listed in the MPS database or has the wrong credentials, MPS will refuse the connection.  Along with this change, we have improved the MPS Devices API to make it very easy to add, edit, and remove devices from the MPS database.  MPS API documentation can be found out on [Swagger](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0#/)

File: docs/release-notes.md [hunk 3]
```diff
@@ -2,51 +2,34 @@
...
     1.	Unplug and re-plug the network cable
     2.	Reboot the Intel&reg; AMT device
 #### UI-Toolkit
 - **KVM freeze intermittently:** We have added a small delay in handling mouse interactions that prevents us from flooding the AMT channel.  There are still a few occasions where KVM could still freeze.  We are still root causing this new issue
-#### MPS
-- **AllowList:** The AllowList is a feature that is supposed to prevent devices from connecting to the MPS unless the GUID is present on the AllowList.  By default, `AllowList = false`.  If you set `AllowList = true` this won't do anything and devices will still be able to connect to MPS, even if they aren't on the AllowList.  
+#### RPS
+- **Failure to add Device:** When RPS is adding new device information to the database and the credentials to Vault, if one of these calls fail (ex. Vault not available), then RPS should revert the changes to the other service and emit an error message.  Currently RPS will provide an error, but it does not revert the changes.  Resolving the failing issue with the service and then re-running the activation will restore the data integrity between the database and Vault.
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:35]: **Failure to add Device:**  If an RPS call to add new device information and credentials to Vault fails, the RPS should revert the changes to the other service and emit an error message. Currently RPS will provide an error, but it does not revert the changes.  Resolving the failing issue with the service and then re-running the activation restores the data integrity between the database and Vault.

**Review by kimberlyxdavis — CHANGES_REQUESTED** [review]

File: docs/release-notes.md (hunk 1 above)
Comment (matt-primrose) [reply docs/release-notes.md:10]: Great changes!  implemented

File: docs/release-notes.md (hunk 2 above)
Comment (matt-primrose) [reply docs/release-notes.md:23]: Updated.

File: docs/release-notes.md (hunk 3 above)
Comment (matt-primrose) [reply docs/release-notes.md:35]: Thank you, more good updates!

**Comment by matt-primrose** [comment]
@kimberlyxdavis I added new section on the Service Event Messages.  Would like your review.

**Review by kimberlyxdavis — APPROVED** [review]

**Pull request merged**